### ISSUE TITLE
any help to update this with ionic RC0

### ISSUE THREAD

**Issue by arsewizz** [ionic2blueprints/ionic2-marketcloud#1]
any help please? i'm using ionic rc0 but i need a help. my test repo [https://github.com/arsewizz/merketCloudRc0Ionic2/](https://github.com/arsewizz/merketCloudRc0Ionic2/)

**Comment by imsingh** [comment]
@arsewizz Yeah. I am working on updating all of the apps to RC1. You will be notified about it. Issue will remain open till then.

**Comment by ironynet** [comment]
When will it be updated to version 2.0 or later?
